stdpopsim/genomes.py

```diff
@@ -386,6 +386,70 @@ def add_dfe(self, intervals, DFE):
         self.dfe_list.append(DFE)
         self.interval_list.append(intervals)
 
+    def add_single_site_mutation_type(
+        self, id, selection_coeff=0.0, dominance_coeff=1.0
```

Comment:
not sure about this default - should be the same as the usual default (which is 0.5: https://github.com/popsim-consortium/stdpopsim/blob/main/stdpopsim/dfe.py#L42 )